doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -858,6 +858,22 @@ change just the hostname via::
   cond %{HEADER:X-Mobile} = "foo"
   set-destination HOST foo.mobile.bar.com [L]
 
+Regex Substitutions
+-------------------
+
+As mentioned above, regex operands are available to match a condition's provided value against
+a regular expression. The substitution elements from these conditional replacement patterns
+can be used in operators within the same header rewriting rule.
+
+For example, ``$ number`` can be used to capture the number group in the replacement
```

Comment:
Is there a max number of capture groups? if so, perhaps `(where number is between 1 and ...)`

Reply:
const int OVECCOUNT = 30; // We support $1 - $9 only, and this needs to be 3x that

Reply:
Thanks for the tip, will add to the doc